src/components/Layout/HomeContent.js

```diff
@@ -396,14 +401,15 @@ export function HomeContent() {
             </div>
             <div className="px-5 lg:px-0 max-w-4xl mx-auto lg:text-center text-secondary dark:text-secondary-dark">
               <Para>
-                With React, you can be a web <i>and</i> a native developer. Your
-                team can ship to many platforms without sacrificing the user
-                experience. Your organization can bridge the platform silos, and
-                form teams that own entire features end-to-end.
+                Com React, você pode ser um desenvolvedor web <i>e</i> nativo.
+                Sua equipe pode enviar para várias plataformas sem sacrificar a
```

Comment:
```suggestion
                Sua equipe pode fazer deploy para várias plataformas sem sacrificar a
```

Apesar de fugir um pouco do original, acho que esse termo é mais popular na comunidade e seria melhor entendido por leitores brasileiros neste contexto.